web/enoent.jsp

```diff
@@ -54,7 +54,7 @@ include file="menu.jspf"
 {
     PageConfig cfg = PageConfig.get(request);
     String configError = "";
-    if (cfg.getSourceRootPath().isEmpty()) {
+    if (cfg.getSourceRootPath() == null || cfg.getSourceRootPath().isEmpty()) {
```

Comment:
Still having troubles understanding how it works/should work. If the call to checkSourceRootExistence() ` in `enoent.jsp` results in exception, this will be handled by `error.jsp` ?

In other words, is the `configError` handling and display of exception needed in `enoent.jsp` ?